scripts/battery_remain.sh

```diff
@@ -20,9 +20,18 @@ pmset_battery_remaining_time() {
 print_battery_remain() {
 	if command_exists "pmset"; then
 		pmset_battery_remaining_time
+    echo "HELLL"
 	elif command_exists "upower"; then
 		battery=$(upower -e | grep battery | head -1)
-		upower -i $battery | grep remain | awk '{print $4}'
+    if is_chrome; then
+      if battery_discharging; then
+        upower -i $battery | grep 'time to empty' | awk '{printf "- %s %s left", $4, $5}'
+      else
+        upower -i $battery | grep 'time to full' | awk '{printf "- %s %s till full", $4, $5}'
+      fi
+    else
+      upower -i $battery | grep remain | awk '{print $4}'
+    fi
```

Comment:
@forkjoseph just push a commit to this same branch and it will be automatically attached to this pull-request.